### PR TITLE
Project improvements

### PR DESCRIPTION
**Pull request by TylerAldrich**
- Updated README to match the instructions we display in the react project
- Updated the code to use generic "Item"/"Items" instead of "Movie"
- Updated CSS/Templates so the look and feel is similar to the react project 
- Removed the instruction about CLI usage section

image

### CODE REVIEW

File: README.md
Comment (tjhiggins) [inline README.md]: Instead of go-demo:latest this should be the architect.yml?

**Review by tjhiggins — CHANGES_REQUESTED** [review]

File: README.md
Comment (TylerAldrich) [reply README.md]: Yep it should be - didn't yet make that update on this proj

**Comment by mueschm** [comment]
Looks good from an engineering perspective and everything works.

**Review by zach-j-dev — APPROVED** [review]
Good from product side

**Comment by zach-j-dev** [comment]
@TylerAldrich good to merge once @tjhiggins approves

**Review by tjhiggins — APPROVED** [review]

**Pull request merged**
